pkg/asset/installconfig/ovirt/ovirt.go

```diff
@@ -0,0 +1,154 @@
+package ovirt
+
+import (
+	"fmt"
+	"github.com/openshift/installer/pkg/types/ovirt"
+	ovirtsdk4 "github.com/ovirt/go-ovirt"
+	"gopkg.in/AlecAivazis/survey.v1"
+	"sort"
+)
+
+// Platform collects ovirt-specific configuration.
+func Platform() (*ovirt.Platform, error) {
+	p := ovirt.Platform{}
+
+	err := survey.Ask([]*survey.Question{
+		{
+			Prompt: &survey.Input{
+				Message: "Enter oVirt's api endpoint URL",
+				Help:    "oVirt engine api url, for example: https://ovirt-engine-fqdn/ovirt-engine/api",
+				Default: "https://rgolan.usersys.redhat.com:8443/ovirt-engine/api",
```

Comment:
I realize this is just the default value, but you probably don't want to have `rgolan.usersys.redhat.com` there.

Reply:
Thanks for noticing, removed it